blinky.md

```diff
@@ -12,11 +12,7 @@ Blinking some lights is the "Hello World" of hardware. Let's make those LEDs bui
 
 In your command line, make a folder for your Tessel code, then initialize a Tessel project in that folder by running each of the following commands in the terminal:
 
-`mkdir tessel-code`
-
-`cd tessel-code`
-
-`t2 init`
+`mkdir tessel-code; cd tessel-code; t2 init`
```

Comment:
"each of the following commands" but now it looks like one command, at least for people not that familiar with the terminal...

How about?

    ```bash
    mkdir tessel-code
    cd tessel-code
    t2 init
    ```